api/models.py

```diff
@@ -1207,6 +1208,14 @@ def __str__(self):
 class FieldReport(models.Model):
     """ A field report for a disaster and country, containing documents """
 
+    class FieldReportStatus(IntegerChoices):
```

Comment:
```suggestion
    class Status(IntegerChoices):
```

This should be fine. Since this is under FieldReport. Also looks good while using :D FieldReport.Status.EW

Reply:
Done.